content/04.network_applications.md

```diff
@@ -14,9 +14,32 @@
 
 #### Deep Learning
 
-1. Define node neighborhoods
-2. Talk about random walks 
-3. Talk about auto encoders random walk independent approaches 
+Deep learning is a paradigm that excels in mapping high dimensional data into a low dimensional space.
+Most techniques under this paradigm have flourished based on a popular natural language processing approach called Word2Vec [@arxiv:1310.4546; @arxiv:1301.3781].
+Word2Vec projects words into a low dimensional space that preserves their semantic meaning.
+This approach uses one of two neural network models, skip-gram and continuous bag of words (CBOW), to accomplish this projection task.
+Both models are feed-forward neural networks that are trained to either predict a word given it's context (CBOW) or the context given a word (skip-gram).
+Once either network has been trained, the finalized network weights are used to represent words and their meaning.
+We discuss approaches that use this central idea to project nodes into a low dimensional space.
+
+Deepwalk [@arxiv:1403.6652] was one of the originating methods to project a knowledge graph into a low dimensional space. 
+First step of this method is to undergo a random walk along a knowledge graph to ascertain how nodes are connected to one another.
+During the random walk, every generated sequence of nodes is recorded and treated like a sentence in the word2vec model [@arxiv:1310.4546; @arxiv:1301.3781].
+After every node has been processed, a skip-gram model is trained to predict the context of each node thereby projecting a knowledge graph into a low dimensional space [@arxiv:1403.6652].
+A caveat to this method is that the random walk takes an unbiased way to traverse each node.
```

Comment:
I don't understand the implications of the caveat - the next sentence might need to tell me what type of applications this hurts.